modvendor/download/download_test.go

```diff
@@ -1162,24 +1162,6 @@ func TestModVendorNoRefFails(t *testing.T) {
 	}, report)
 }
 
-func TestModVendorVendorDirIsRelativeFails(t *testing.T) {
```

Comment:
now it's impossible to have vendorDir as a relative path, so this test doesn't make sense anymore.